richtext-commonmark/build.gradle

```diff
@@ -8,8 +8,6 @@ dependencies {
   compileOnly deps.compose.tooling
 
   api project(":richtext-ui")
-  api deps.compose.foundation
```

Comment:
I'm not sure about this but wouldn't this change also disable library consumers to silently import `foundation` and `text` libraries because now `api` visibility is gone? I'm asking this since `richtext-ui` enables consumers to have access to those and if this module is exposing `richtext-ui`, then its public dependencies shouldn't be buried here. However, I might be totally wrong on this since I never tried it. Maybe just having `api project(":richtext-ui")` is enough to expose the same dependencies.

Reply:
nope, just tested this out. I'm totally wrong. It's all safe to remove these dependencies.

Reply:
Oh, i was just trying to elaborate more but figured you already get it